### DISCUSSION TITLE
Dealing with "License Limit" Warnings during Server Migrations

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5518] [FAQs]
Hello everyone!

I've been working on server migration recently, and when I entered the AMS license key under Settings on the Web Panel for the new instance, it showed a warning message

> You have reached your license limit. Please close some of your running instances

How can I fix this?

Thank you

**Comment by muratugureminoglu** [comment]
Hi @Mohit-3196 

This means that you have exceeded the license usage limit (If you are not using a Monthly license). To resolve this, you should shut down your active Ant Media Server instances and try again after 15 minutes. If you continue to experience issues, please contact support.

**Reply by mekya** [reply]
I think it should not give this error for monthly licenses anymore because it just calculates over usage. Please correct me if I'm wrong

**Reply by muratugureminoglu** [reply]
Yes, you are correct, Thanks for the addition.
